server/tokenRegistrySubmitter.js

```diff
@@ -0,0 +1,49 @@
+require('isomorphic-fetch')
```

Comment:
Is this really a "submitter"? From what I gather "proxy" would be a more accurate description

Reply:
Refactored